image-layout.md

```diff
@@ -5,7 +5,7 @@ This layout MAY be used in a variety of different transport mechanisms: archive
 
 Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md) by:
 
-* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [manifest list](manifest-list.md#manifest-list)
+* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [image index](image-index.md#manifest-list)
```

Comment:
> this is like the Image Layout, no?

Yeah, we should probably rename that too ;).

The only thing particularly image-specific about image-spec is the config format, and when we have sufficient context, we refer to that as “[a configuration][1]”.  The full name (for when you don't have sufficient context) is “[OCI Image Configuration][2]”.  I think that's a good pattern to follow, and it would mean we'd have something like the following formats:

| Current | Short | Long |
| -- | -- | -- |
| image layout | directory storage | OCI Image Directory Storage |
| manifest list | index | OCI Image Index |
| manifest | manifest | OCI Image Manifest |
| configuration | configuration | OCI Image Configuration |
| layer (and "[layer filesystem changeset](https://github.com/opencontainers/image-spec/blob/v1.0.0-rc4/layer.md#image-layer-filesystem-changeset)”) | layer | OCI Image Layer |

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/spec.md#L3
[2]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc4/config.md#oci-image-configuration